patches/minecraft/net/minecraft/world/inventory/AbstractContainerMenu.java.patch

```diff
@@ -12,6 +12,14 @@
           for(ContainerListener containerlistener : this.f_38848_) {
              containerlistener.m_7934_(this, p_150408_, itemstack1);
           }
+@@ -399,6 +_,7 @@
+             ItemStack itemstack10 = slot7.m_7993_();
+             ItemStack itemstack11 = this.m_142621_();
+             p_150434_.m_141945_(itemstack11, slot7.m_7993_(), clickaction);
++            if (!net.minecraftforge.common.ForgeHooks.onItemStackedOn(itemstack11, itemstack10, slot7, clickaction, p_150434_, m_150446_()))
```

Comment:
The core team would like the hook to be moved *after* the vanilla method calls, to give the item priority.

Reply:
What about the case of denying vanilla behavior? If Orion has an issue he can explain himself here.

Reply:
I already did.
1) my mod has custom functionality that this event can majorly break
2) forge has an unwritten rule not to make systems which allow one mod to break another of if can be prevented
3) there is currently no vanilla behavior, the methodbodies are empty, if you are talking about the bundle: that is not officially part of the game, but would need a special event anyway.
4) this was requested by the core team.

Do you need reasons why this needs to be changed and is rejected as of now?

Reply:
I can accept this explanation. Sorry if I caused any stress, just prefer things to be conducted in the open.